tests/appsec/iast/source/test_uri.py

```diff
@@ -0,0 +1,16 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+from utils import coverage
+from .._test_iast_fixtures import BaseSourceTest
+
+
+@coverage.basic
+class TestURI(BaseSourceTest):
+    """Verify that URL is tainted"""
+
+    endpoint = "/iast/source/uri/test"
+    requests_kwargs = [{"method": "GET"}]
+    source_type = "http.request.uri"
+    source_value = None
```

Comment:
The `source_value` should be checked. I assume it should be something like `http://localhost:7777/iast/source/uri/test`.

Reply:
Fixed